### PR TITLE
Feature/vih 4858 representative optout journey

### PR DESCRIPTION
**Pull request by kkodalikk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-4858

### Change description ###

Representative optout journey changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <h1 class="govuk-heading-xl" i18n="@@otherInformation_h1">
+      Other information
+    </h1>
+    <div>
+      <app-hearing-details-header></app-hearing-details-header>
+    </div>
+
+
+
+    <app-choice-textbox [detailsYesText]="'Please provide any information here.'"
```
Comment (valentinabennett) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html]: need set i18n

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                        [suitabilityAnswer]="model.otherInformation"
+                        (submitted)="submit()">
+
+      <p class="govuk-body govuk-!-margin-bottom-6" i18n="@@aboutYou_p_1">
+        Is there anything you'd like the court to know that could affect this hearing taking place by video?
```
Comment (valentinabennett) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html]: i18n should be @@otherInformation...

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html (hunk 1 above)
Comment (kkodalikk) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html]: Added

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html (hunk 2 above)
Comment (kkodalikk) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/other-information/other-information.component.html]: done

**Pull request merged**